### PR TITLE
Update requirements.txt

### LINKED ISSUE #1: Information Exposure vulnerability in requests dependency
The `requests` package before `2.20.0` sends an HTTP Authorization header to an HTTP URI upon receiving a same-hostname https-to-http redirect, which makes it easier for remote attackers to discover credentials by sniffing the network.

**More info**
[Snyk](https://app.snyk.io/vuln/SNYK-PYTHON-REQUESTS-72435)
[CVE-2018-18074](https://nvd.nist.gov/vuln/detail/CVE-2018-18074)

**Remediation**
Upgrade requests to version `2.20.0` or later.
```
requests>=2.20.0
```

### PR DESCRIPTION
**Pull request by rbricheno**
The requests package before 2.20.0 sends an HTTP Authorization header to an HTTP URI upon receiving a same-hostname https-to-http redirect, which makes it easier for remote attackers to discover credentials by sniffing the network. Update requirements.txt to spcify a version of at least 2.20.0 for requests

This should close #1

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1 +1 @@
-requests>=2.18.0,<2.18.99
+requests>=2.20.0
```
Comment (iamtraction) [inline requirements.txt]: We shouldn't auto install/update major releases for the package.
```suggestion
requests>=2.20.0,<3.0.0
```

**Review by iamtraction — CHANGES_REQUESTED** [review]
Just one little change and this will be ready to merge.

**Comment by rbricheno** [comment]
Thanks, updated.

**Review by iamtraction — APPROVED** [review]
Thank you for your contribution! We would like to thank you by **inviting you to join the PokeDevs organization on GitHub.** Accept the invite by visiting https://github.com/orgs/PokeDevs/invitation.

Thanks again!

**Pull request merged**